toffy/image_stitching.py

```diff
@@ -87,7 +88,7 @@ def stitch_images(tiff_out_dir, run_dir=None, channels=None, img_sub_folder=None
 
     # get load and stitching args
     num_cols = math.isqrt(len(folders))
-    max_img_size = get_max_img_size(tiff_out_dir, run_dir)
+    max_img_size = get_max_img_size(tiff_out_dir, img_sub_folder, run_dir)
```

Comment:
Is there anywhere else in the repo that this is used?

Reply:
Just the rosetta notebook, but that isn't set up to use a FOV subdirectory structure anyways.